Trackovid19-web/src/app/shared/share-location-status/share-locations-status-modal.component.html

```diff
@@ -0,0 +1,56 @@
+<app-basic-modal
```

Comment:
this is related with the other PR right?

Reply:
yup, removed!